posters-overrides.json

```diff
@@ -2954,7 +2954,13 @@
    "number": 1056
   },
   {
-   "number": 1057
+   "number": 1057,
+   "title": "AxonDeepSeg: Automatic Myelin and Axon Segmentation Using Deep Learning",
+   "institution": "Montreal Heart Institute",
+   "presenter": "Mathieu Boudreau",
+   "categories": "Modeling and Analysis Methods<br>Segmentation and Parcellation",
+   "videochat": "<a href=\"https://meet.jit.si/ohbm2020-1057\" target=\"_ohbm2020-1057\">jitsi:ohbm2020-1057</a>",
```

Comment:
If these aren't changes from `posters.json`, they don't need to be specified.

Reply:
I will mege an prune